src/utilities.ts

```diff
@@ -215,6 +215,11 @@ export function getProviderName(provider: any): string | undefined {
     return 'MYKEY'
   }
 
+  if (provider.isHuobiWallet) {
```

Comment:
```suggestion
  if (provider.isHbWallet) {
```

During my testing on iOS, this is where the boolean flag namespace was. Is this the same for all devices?